lib/kitchen/transport/winrm.rb

```diff
@@ -137,6 +137,15 @@ def upload(locals, remote)
           file_transporter.upload(locals, remote)
         end
 
+        # (see Base::Connection#download)
+        def download(remotes, local)
+          # This looks possible with WinRM::FS::FileManager
```

Comment:
you could probably do this reasonably trivially - https://github.com/chef/train/commit/8fb6a6bc71f26c8cbaacf977a55ff827fca25240#diff-f2edeefbb087259b847aa6f5ded7d0f4 is the corresponding work in train.

Reply:
I added the download and file_manager methods from there.